docs/content/docs/connectors/datastream/formats/text_files.md

```diff
@@ -0,0 +1,69 @@
+---
+title:  "Text files"
+weight: 4
+type: docs
+aliases:
+- /dev/connectors/formats/text_files.html
+- /apis/streaming/connectors/formats/text_files.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+
+# Text files format
```

Comment:
I don't have a better suggestion, but I think `Text files format` is ambiguous. It could be CSV, JSON, AVRO etc. WDYT?

Reply:
I think text is better as this format provides `Strings` that can be parsed as CSV, JSON etc... in a map downstream the pipeline.